client/README.md

```diff
@@ -100,6 +100,9 @@ func main() {
 	if err := c.Write(bp); err != nil {
 		log.Fatal(err)
 	}
+	
+	// Close client resources
+	c.Close()
```

Comment:
I think this code would be better. On line 75, add this:

```golang
defer c.Close()
```

This ensures that close is always called.

At the end, instead, do this:

```golang
if err := c.Close(); err != nil {
    log.Fatal(err)
}
```

Looking at the code, this can't fail, but it's a good practice and the Close method should be capable of being called multiple times. In case we ever needed to modify the code to report some kind of failure (unlikely), it's good to be able to handle it.